packages/app/src/components/catalog/EntityPage.tsx

```diff
@@ -222,6 +228,32 @@ const chaosContent = (
     </EntitySwitch.Case>
   </EntitySwitch>
 );
+
+const ccmContent = (
+  <EntitySwitch>
+    <EntitySwitch.Case if={isHarnessCcmAvailable}>
+      <EntityCcmContent />
+    </EntitySwitch.Case>
+
+    <EntitySwitch.Case>
+      <EmptyState
+        title="No Cloud Cost Management data available for this entity"
+        missing="info"
+        description="You need to add an annotation to your component if you want to enable Cloud Cost Management for it. You can read more about annotations in Backstage by clicking the button below."
```

Comment:
Can we add the annotation info here? `harness.io/perspective-url`

Reply:
Thanks for the feedback! I have made the requested changes.